src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -406,8 +409,8 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
     this.setNameRequest({
       legalType: this.getEntitySnapshot?.businessInfo?.legalType,
       legalName: this.getEntitySnapshot?.businessInfo?.legalName,
-      nrNumber: this.getEntitySnapshot?.businessInfo?.nrNumber
-    })
+      nrNum: this.getEntitySnapshot?.businessInfo?.nrNumber
+    } as any)
```

Comment:
Legal API calls this `nrNumber` but Namex calls this `nrNum`.

You'll see conversion between Legal API and store a lot in filing-template-mixin.ts.

Need to cast to "any" because this isn't a complete NR object (missing properties but we don't need them here).